VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -0,0 +1,160 @@
+import { StackScreenProps } from '@react-navigation/stack'
+import { map } from 'underscore'
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { BackgroundVariant, BorderColorVariant, Box, BoxProps, ClickToCallPhoneNumber, LargePanel, TextView, VAIcon } from 'components'
+import { BranchesOfServiceConstants, ServiceData } from 'store/api/types'
+import { DisabilityRatingState, MilitaryServiceState, PersonalInformationState } from 'store/slices'
+import { HomeStackParamList } from '../HomeStackScreens'
```

Comment:
We typically try to avoid relative imports

Reply:
fixed.